test/dashboard-e2e/.gitignore

```diff
@@ -0,0 +1,2 @@
+videos/
```

Comment:
I believe we should also include screenshots folder to gitignore

Reply:
updated